cfme/tests/configure/test_default_views.py

```diff
@@ -84,3 +101,33 @@ def test_grid_defaultview(request, setup_a_provider, key):
     sel.force_navigate(name[1])
     assert tb.is_vms_grid_view(), "Grid Default view setting failed"
     reset_default_view(name[0], default_view)
+
+
+@pytest.mark.parametrize('key',
+    cfme_data.get('defaultview_data', {})['exp_comp']['infra'], scope="module")
+def test_expanded_view(request, setup_a_provider, key):
+    name = re.split(r"\/", key)
```

Comment:
Using re.split seems like overkill here. I'd expect `name = key.split('/')` to do the job just as well without needing a regex engine. That said, since we're pulling this from yaml, I'd much rather see us implement this structure in yaml, rather than having to parse the yaml after we've already parsed it into python.

```yaml
    exp_comp:
            infra:
                - Compare/infrastructure_hosts/Configuration/Compare Selected Hosts
```

should be

```yaml
    exp_comp:
            infra:
                - [Compare, infrastructure_hosts, Configuration, Compare Selected Hosts]
```

Once the yaml is tweaked, I think value unpacking would help with readability, like `name, page, button, button_child = key` (probably changing the name of `key` to something that describes its new role a little better)